File: clover/src/app/entities/Seat/seat.ts
export class Seat {
id:number;
number2:number;
class2:string;
price:number;
discount:number;
taken:boolean;
flightInfo2Id:number;

constructor( number: number, class2: string,price :number,discount :number,taken :boolean,flightId:number){

this.class2 = class2;
this.number2 = number;
this.price = price;
this.taken = false;
this.discount = 0;
}
}

File: Backend/AirlineAPI/Models/Seat.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace AirlineAPI.Models
{
    public class Seat
    {
        [Key]
        public int Id { get; set; }
        [Required]
        public int Number2 { get; set; }
        [Required]
        public string Class2 { get; set; }
        [Required]
        public int Price { get; set; }
        [Required]
        public int Discount { get; set; }
        [Required]
        public bool Taken { get; set; }

      //  [ForeignKey("FlightInfo2")]
        public int FlightInfo2Id { get; set; }
        //public FlightInfo2 FlightInfo2 { get; set; }
    }
}
